src/charactersheet/viewmodels/character/items/view.html

```diff
@@ -0,0 +1,30 @@
+
+<card-edit-actions params="{ flip: $parent.flip }"></card-edit-actions>
+<div data-bind="with: entity">
+    <div class="h4 collapsable-card-title"
+        data-bind="text: name" />
+    <div class="panel-body row">
+    <div class="col-sm-5 col-md-4 col-lg-3">
+        <table class="table table-hover table-condensed table-ac-bordered">
+        <tbody>
+            <tr>
+            <th>Quantity:</th>
+            <td><span data-bind="text: quantity"></span></td>
+            </tr>
+            <tr>
+            <th>Weight Each:</th>
```

Comment:
Should this say `Container Weight` or just `Weight` when this is a container? Since containers cant be multiple